scribble-lib/scribble/manual-racket.js

```diff
@@ -236,12 +236,15 @@ function AddPartTitleOnClick(elem) {
         else
             elem.parentNode.appendChild(info);
 
-        /* Clicking the header shows the explanation element: */
-        elem.onclick = function () {
-            if (info.style.display == "none")
+        /* Clicking the information button shows the explanation element: */
+        const heading = elem.querySelector('.heading-source');
+        if (heading) {
```

Comment:
Why is this test needed? Can it ever fail?

Reply:
Technically, yes. If users insert, say, an `h3` element with custom attributes that happen to match the structure that we are looking for, but did not have the corresponding `.heading-source` button, then `heading` would be `null`.